elastic4s-core/src/main/scala/com/sksamuel/elastic4s/requests/indexes/MappingHandlers.scala

```diff
@@ -15,7 +15,7 @@ trait MappingHandlers {
           val raw = ResponseHandler.fromResponse[Map[String, Map[String, Map[String, Any]]]](response)
           val raw2 = raw.map {
             case (index, types) =>
-              val mappings = types("mappings").getOrElse("properties", Map.empty)
+              val mappings = types("mappings").getOrElse("properties", types("mappings"))
```

Comment:
`GetAllMappings` has an output with "properties"
```
{
    "index": {
        "mappings": {
            "properties": {
                "first_attribute": {
```

But when you look by fields, it doesn't have the properties field. It looks like:

```
{
    "index": {
        "mappings": {
            "first_attribute": {
```

I tested on 7.1.1 

To be frank, I'm not sure in what circumstance we'd expect properties to not be there in `GetAllMappings` so I removed the empty map.